components/suggestion/emoticon_provider.jsx

```diff
@@ -90,6 +90,15 @@ export default class EmoticonProvider extends Provider {
         return emojis.map((item) => ':' + item.name + ':');
     }
 
+    // findAndSuggestEmojis uses the provided partialName to match anywhere inside an emoji name.
+    //
+    // For example, typing `:welc` would match both `:welcome:` and `:youre_welcome:` if those
+    // emojis are present in the local store. Note, however, that the server only does prefix
+    // matches, so a query to populate the local store for `:welc` would only return `:welcome:`.
+    // This results in surprising differences between a fresh load of the application, and the
+    // changes to the cache from expanding the cache with emojis found in existing posts.
+    //
+    // For now, this behaviour and difference is by design. @see MM-17320.
```

Comment:
```suggestion
    // For now, this behaviour and difference is by design. See MM-17320.
```

Reply:
^unless `@see` is the convention for comments

Reply:
0/5 We may also want to link to the ticket instead of supplying a ticket number, to make it easier for non-Mattermost staff to find it.